t/lib/t/MusicBrainz/Server/Controller/Edit/Show.pm

```diff
@@ -12,7 +12,7 @@ my $hard_mock_edit_class = $mock_edit_class + 1;
     package t::Controller::Edit::Show::MockEdit;
     use Moose;
     extends 'MusicBrainz::Server::Edit';
-    sub edit_name { 'Add artist' } # Just so we use an edit template
+    sub edit_name { 'Edit artist' } # Just so we use an edit template
```

Comment:
At some point we'll need to change this further, but for now I'm changing it to a template with diffs since I know I won't be changing those soon.